ecs-cli/modules/cli/compose/entity/entity_test.go

```diff
@@ -58,6 +58,36 @@ func TestGetContainersForTasks(t *testing.T) {
 	assert.Equal(t, containers[0].Ec2IPAddress, aws.StringValue(ec2Instance.PublicIpAddress), "Expects PublicIpAddress to match")
 }
 
+func TestGetContainersForTasksPrivateIP(t *testing.T) {
+	containerInstanceArn := "containerInstanceArn"
+	ec2InstanceID := "ec2InstanceId"
+	ec2Instance := &ec2.Instance{PrivateIpAddress: aws.String("publicIpAddress")}
+
+	ecsTasks := []*ecs.Task{
+		&ecs.Task{
+			Containers: []*ecs.Container{
+				&ecs.Container{
+					Name: aws.String("containerName"),
+				},
+			},
+			ContainerInstanceArn: aws.String(containerInstanceArn),
+		},
+	}
+	containerInstancesMap := make(map[string]string)
+	containerInstancesMap[containerInstanceArn] = ec2InstanceID
+
+	ec2InstancesMap := make(map[string]*ec2.Instance)
+	ec2InstancesMap[ec2InstanceID] = ec2Instance
```

Comment:
Naming nit: Generally, I try to avoid having the name of the data structure in the variable, so maybe just `ec2Instances`? Also, in spite of the fact that it's the actual term in ECS, I find containerInstances to be a little verbose -- perhaps just `ecsInstances`?

Reply:
I'll remove the type from the variable. This is done in like a billion variables in the test file... so I'll do it everywhere. I'm going to stick with containerInstances since its a real ECS term and its used so many places in this test file.